frontend/src/data/variables/CdcCovidProvider.test.ts

```diff
@@ -34,6 +24,38 @@ const METRIC_IDS: MetricId[] = [
   "covid_cases_reporting_population_pct",
 ];
 
+export async function ensureCorrectDatasetsDownloaded(
+  cdcDatasetId: string,
+  acsDatasetIds: string[],
+  baseBreakdown: Breakdowns,
+  breakdownVar: BreakdownVar,
+  rawCovidData: any,
+  finalCovidData: any
+) {
+  const acsProvider = new AcsPopulationProvider();
+  const cdcCovidProvider = new CdcCovidProvider(acsProvider);
+
+  var consumedDatasetIds = [];
```

Comment:
is there a specific reason for `var` here? For React I think the general preference (and the way most of the code base is currently) is to prefer `const` or `let` over `var` as they are block scoped. `const` is usually the go to, and using `let` only if you need to mutate the variable. In this case `const` is probably preferred since we aren't re-assigning the array itself, only adding items to it.

Reply:
sounds good